### ISSUE TITLE
Intention prediction 

### ISSUE THREAD

**Issue by munirfarzeen** [vita-epfl/bounding-box-prediction#16]
Hi,
I am running your code with JAAD dataset. Without making any changes in the code, when I calculate F1 score for Intention prediction and state prediction it is basically zero and the same values of Intention accuracy (0.8717)and state accuracy(0.9257) keep repeating. 
![image](https://github.com/vita-epfl/bounding-box-prediction/assets/23066497/58023460-251c-45fd-ae34-e0c6e5758ab9)
![image](https://github.com/vita-epfl/bounding-box-prediction/assets/23066497/5381da6a-5598-44f3-932f-db940a4398e5)

Can you see, why is it like that?
Thank you

**Comment by Yachilan** [comment]
Hello,
Honestly, we never used those metrics during the project. We successfully recreated the error and found a possible fix. In the `sklearn.metrics` functions replace the `average=binary` with `average=micro`.
See also the documentation of [sklearn.f1_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html#sklearn.metrics.f1_score)
